packages/cli/src/base.ts

```diff
@@ -167,22 +170,37 @@ export abstract class BaseCommand extends Command {
     return this._kit
   }
 
+  private async getTransport(): Promise<Transport> {
+    const nodeUrl = await this.getNodeUrl()
+    return nodeUrl && nodeUrl.endsWith('.ipc') ? ipc(nodeUrl) : http(nodeUrl)
+  }
+
+  private async openLedgerTransport() {
+    if (!this.ledgerTransport) {
+      // types seem to be suggesting 2 defaults but js is otherwise for TransportNodeHid
+      const TransportNodeHid: typeof _TransportNodeHid =
+        // @ts-expect-error
+        _TransportNodeHid.default || _TransportNodeHid
+      this.ledgerTransport = await TransportNodeHid.open('')
+    }
+    return this.ledgerTransport!
+  }
+
   // TODO(viem): This shouldn't be public, but for the time being to be called
   // from CheckBuilder to allow smooth transitions it is public.
   public async getPublicClient(): Promise<CeloClient> {
     if (!this.publicClient) {
       const nodeUrl = await this.getNodeUrl()
-
-      const transport = nodeUrl && nodeUrl.endsWith('.ipc') ? ipc(nodeUrl) : http(nodeUrl)
+      const transport = await this.getTransport()
 
       // Create an intermediate client to get the chain id
       const intermediateClient = createPublicClient({
         transport,
       })
       const chainId = await intermediateClient.getChainId()
       const extractedChain = extractChain({
-        chains: [celo, celoAlfajores, celoBaklava],
```

Comment:
the fallback should work for baklava anyway. we can do a quick manual test to be sure

Reply:
Oh oops I forgot to undo this change